manager/allocator/network.go

```diff
@@ -696,6 +714,32 @@ func (a *Allocator) allocateTask(ctx context.Context, nc *networkContext, tx sto
 	return storeT, nil
 }
 
+func (a *Allocator) procUnallocatedNetworks(ctx context.Context, nc *networkContext) {
+	for _, n := range nc.unallocatedNetworks {
+		if !nc.nwkAllocator.IsAllocated(n) {
+			if err := a.allocateNetwork(ctx, nc, n); err != nil {
+				log.G(ctx).Errorf("Failed allocation of unallocated network %s: %v", n.ID, err)
```

Comment:
Yeah it's possible if we get into that situation where there is a lot of state changes. May be move it to `Info` or Debug level?